build_run_docker.sh

```diff
@@ -219,17 +240,29 @@ fi
 
 # We use this for the docker-compose project name, which will not allow "."
 RONDB_VERSION_NO_DOT=$(echo "$RONDB_VERSION" | tr -d '.')
+# We can't put an arbitrary command line or arbitrary list of packages as part
+# of a filename, so hash them. User can refer to parsed_arguments.txt for full
+# context.
+MYSQLD_INSTRUMENTATION_INDICATOR=
+if [ -n "$MYSQLD_INSTRUMENTATION" ]; then
+    MYSQLD_INSTRUMENTATION_INDICATOR="$(echo "$MYSQLD_INSTRUMENTATION" | sha256sum | sed -r 's/^(.{8}).*$/_mi-\1/')"
```

Comment:
I think 3 characters is already enough? 256x256x256=16m combinations

Reply:
The hash sum is in hexadecimal, so if you want 24 bits of information then that's 6 characters.

Reply:
Ah right, okay. 16^4 is okay (65k)? I don't expect ppl to try huge amounts of different combinations without deleting the folders in-between.

Reply:
(Sorry, I just get annoyed by the long folder names in my IDE 😅)

Reply:
Agreed, with 16⁴ hashes we'd expect a collision after about √ (16⁴*π/2)≈320 evaluations, and if that many are used one's likely to try to parallelize anyway, and then we have a clash in container names instead.

Reply:
Fixed in `ee54619`